test/suites/ACID.m

```diff
@@ -2434,6 +2435,26 @@
     set(ax4, 'Color', 'none');
 end
 % =========================================================================
+function [stat] = histogramPlot()
+
+  if ~exist('histogram')
```

Comment:
I'm not happy with `~exist('histogram')` because this is way too broad. Just creating a file called "histogram" would pass that test, but would make the code after that break.
IMHO, the best way is `isempty(which('histogram'))` because this directly checks for the existence of such a function.